ArcGISRuntimeSDKQt_CppSamples/Scenes/GetElevationAtPoint/GetElevationAtPoint.h

```diff
@@ -73,7 +70,7 @@ private slots:
   bool elevationQueryRunning() const;
 
   double m_elevation = 0.0;
-  Esri::ArcGISRuntime::TaskWatcher m_elevationQueryTaskWatcher;
+  bool m_elevationQueryRunning = false;
```

Comment:
Can you store the Future instead? It also has an isRunning API that would be meaningful.